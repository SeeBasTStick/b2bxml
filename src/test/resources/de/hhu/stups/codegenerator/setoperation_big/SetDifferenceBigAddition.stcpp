int main() {
    SetDifferenceBig set;
    set.simulate();
    cout << set.getSet().card();
    return 0;
}