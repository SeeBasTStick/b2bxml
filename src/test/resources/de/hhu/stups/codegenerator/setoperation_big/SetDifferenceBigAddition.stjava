public static void main(String[] args) {
    SetDifferenceBig set = new SetDifferenceBig();
    set.simulate();
    System.out.println(set.set.card());
}