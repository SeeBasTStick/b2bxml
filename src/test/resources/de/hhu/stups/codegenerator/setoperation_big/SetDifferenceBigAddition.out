public static void main(String[] args) {
    SetComplementBig set = new SetComplementBig();
    set.simulate();
    System.out.println(set.set);
}